state/app/DataManager.kt

```diff
@@ -114,6 +117,18 @@ class DataManager {
         var graphOutputEnabled: Boolean?
             get() = properties?.getProperty(GRAPH_OUTPUT)?.toBoolean()
             set(value) = setProperty(GRAPH_OUTPUT, value.toString())
+
+        var graphTheme: Color.GraphTheme?
+            get() = stringToTheme(properties?.getProperty(GRAPH_THEME))
+            set(value) = setProperty(GRAPH_THEME, value!!.name.lowercase())
+
+        private fun stringToTheme(themeString: String?): Color.GraphTheme? {
+            return when (themeString) {
+                Label.DARK_GRAPH_NAME.lowercase() -> Color.Themes.DARK_GRAPH
+                Label.LIGHT_GRAPH_NAME.lowercase() -> Color.Themes.LIGHT_GRAPH
```

Comment:
Our graph name labels are formatted as `Dark`, but it feels as though it makes more sense for them to be in lowercase in a properties/config file.